style.md

```diff
@@ -141,6 +141,9 @@ resources:
 2. [Go Common Mistakes](https://github.com/golang/go/wiki/CommonMistakes)
 3. [Go Code Review Comments](https://github.com/golang/go/wiki/CodeReviewComments)
 
+All code samples are based on the two most recent major versions of Go
```

Comment:
minor. 1.18 and 1.17 are minor releases.
```suggestion
All code samples are based on the two most recent minor versions of Go
```

Also, maybe "We aim for code samples to be accurate for the two most recent minor [..]".
Since we sometimes fall behind?

Reply:
👍 will update